openmetadata-spec/src/main/resources/json/schema/api/data/createTopic.json

```diff
@@ -80,6 +80,10 @@
       "description": "Entity extension data with custom attributes added to the entity.",
       "$ref": "../../type/basic.json#/definitions/entityExtension"
     },
+    "sourceUrl": {
+      "description": "Source URL of topic.",
+      "$ref": "../../type/basic.json#/definitions/sourceUrl"
```

Comment:
@07Himank we already have URL type, take a look at that

Reply:
@harshach I took reference from dashboard.json , where sourceURL was already there do we need to update that ?